src/actions/helpers/index.ts

```diff
@@ -1,9 +1,32 @@
+import { makeApiCall } from "../../api/helpers";
 import { Registrar } from "../../contracts/types";
-
 import { getLogger } from "../../utilities";
+import {
+  ETHEREUM_API_ID,
+  ETHEREUM_PRICE_ENDPOINT,
+  ERC20_PRICE_ENDPOINT,
+} from "./constants";
+import { 
+  CoinGeckoRequestOptions,
+  CoinGeckoResponse,
+  CoinGeckoPrice
+} from "../../types";
 
 const logger = getLogger("actions:helpers");
 
+const createUri = (tokenAddress?: string): string => {
+  const options: CoinGeckoRequestOptions = {
+    ids: tokenAddress ?? "ethereum",
```

Comment:
To clarify, this could be "ethereum" or the token address (ie: `0x1234....`)?

Reply:
Yes, the id is either an ERC20 token or we are requesting the price for ethereum itself. This can be a bit better though because that value could just be hard coded into the existing constant for that endpoint anyways